providers/provider_data.go

```diff
@@ -41,6 +41,7 @@ type ProviderData struct {
 
 	// Common OIDC options for any OIDC-based providers to consume
 	AllowUnverifiedEmail bool
+	UserClaim            string
```

Comment:
This stuff was removed from the GitLab PR (because Nickname isn't a claim in the minimal IDToken apparently, its API only).

But I'm tempted to leave this in because it is handy to consume by others providers embracing this pattern (embedding OIDCProvider).